en/getting_started/README.md

```diff
@@ -59,14 +59,15 @@ You **need to install Qt as described below** instead of using pre-built package
 To install Qt:
 
 1. Download and run the [Qt Online Installer](http://www.qt.io/download-open-source)
-   - **Ubuntu:** 
-     - Set the downloaded file to executable using: `chmod +x`. 
+   - **Ubuntu:**
+     - Set the downloaded file to executable using: `chmod +x`.
      - Install to default location for use with **./qgroundcontrol-start.sh.** If you install Qt to a non-default location you will need to modify **qgroundcontrol-start.sh** in order to run downloaded builds.
+   - **Arch Linux:** Run the [setup script](https://github.com/mavlink/qgroundcontrol/blob/master/tools/setup/arch.sh) for all dependiencies **including Qt**.
```

Comment:
@DonLakeFlyer @MaEtUgR 
Don, saw your comments on this and https://github.com/mavlink/qgroundcontrol/pull/9386

So your argument is that this line should be removed - the script will pull the wrong component; so you should just use Qt online installer. Right?

Note, if this does stay here, we need to fix typo "dependiencies"

Reply:
That is correct

Reply:
Let's settle the discussion at https://github.com/mavlink/qgroundcontrol/pull/9386#discussion_r577069689 and I'll either correct the typo or revert the addition altogether.

Reply:
Discussion settled: https://github.com/mavlink/qgc-dev-guide/pull/124, https://github.com/mavlink/qgroundcontrol/pull/9477